app/resources/api/v2/plate_resource.rb

```diff
@@ -21,26 +47,74 @@ class PlateResource < BaseResource
       # labware
       include Api::V2::SharedBehaviour::Labware
 
-      # Constants...
-
+      # TODO: {Y24-213} Possibly this line doesn't actually do anything and could be removed.
       default_includes :uuid_object, :barcodes, :plate_purpose, :transfer_requests
```

Comment:
Its name is misleading as well. In older versions of the library, it was used for pre-loading associations. It sounds like the include parameter to make a compound document, but it is not.